python/tchannel/event.py

```diff
@@ -40,11 +40,9 @@
     after_receive_request=0x21,
     before_receive_response=0x30,
     after_receive_response=0x31,
-    after_receive_system_error=0x40,
-    after_receive_system_error_per_attempt=0x41,
-    after_send_system_error=0x42,
-    on_operational_error=0x50,
-    on_operational_error_per_attempt=0x51,
```

Comment:
Are these hooks being replaced and consolidated? Add relevant entries to the changelog: "Replace X and Y hooks with Z." Also, this counts as backwards breaking if these hooks were present in the last released version. You'd want to bump the major version.